File: lanjii/lanjii-admin-ui/src/types/log/sysLoginLog.ts
/** 系统登录日志 */
export interface SysLoginLog {
  /** 日志ID */
  id?: number
  /** 用户名 */
  username?: string
  /** 登录IP地址 */
  ipAddress?: string
  /** 登录地点 */
  loginLocation?: string
  /** 浏览器类型 */
  browser?: string
  /** 操作系统 */
  os?: string
  /** 登录类型（0-登录，1-登出） */
  loginType?: number
  /** 登录状态（0-失败，1-成功） */
  status?: number
  /** 提示消息 */
  msg?: string
  /** 登录时间 */
  loginTime?: string
  /** 创建时间 */
  createTime?: string
}

File: lanjii/lanjii-modules/module-system/system-biz/src/main/java/com/lanjii/sys/entity/SysLoginLog.java
package com.lanjii.sys.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.lanjii.system.api.dto.SysLoginLogDTO;
import com.lanjii.system.api.vo.SysLoginLogVO;
import com.lanjii.framework.mp.base.TenantBaseEntity;
import com.lanjii.sys.base.AdminBaseMapper;
import lombok.Data;
import lombok.EqualsAndHashCode;
import org.mapstruct.Mapper;
import org.mapstruct.factory.Mappers;

import java.time.LocalDateTime;

/**
 * 用户登录日志表(SysLoginLog)实体类
 *
 * @author lanjii
 */
@EqualsAndHashCode(callSuper = true)
@Data
@TableName("sys_login_log")
public class SysLoginLog extends TenantBaseEntity {

    /**
     * 日志ID
     */
    private Long id;

    /**
     * 用户名
     */
    private String username;

    /**
     * 登录IP地址
     */
    private String ipAddress;

    /**
     * 登录地点
     */
    private String loginLocation;

    /**
     * 浏览器类型
     */
    private String browser;

    /**
     * 操作系统
     */
    private String os;

    /**
     * 登录类型（0-登录，1-登出）
     */
    private Integer loginType;

    /**
     * 登录状态（0-失败，1-成功）
     */
    private Integer status;

    /**
     * 提示消息
     */
    private String msg;

    /**
     * 登录时间
     */
    private LocalDateTime loginTime;

    @Mapper
    public interface SysLoginLogMapper extends AdminBaseMapper<SysLoginLog, SysLoginLogVO, SysLoginLogDTO> {

    }

    public static final SysLoginLogMapper INSTANCE = Mappers.getMapper(SysLoginLogMapper.class);

}
